source/launch.txt

```diff
@@ -24,13 +24,23 @@ databases. You have several options for creating a schema and
 launching :binary:`~bin.mongosqld`. This guide will help you choose
 the best option for your needs.
 
-By default, :binary:`~bin.mongosqld` generates a :doc:`data schema
-</schema-configuration>` and holds it in memory. Alternatively, if
-you prefer to create a schema file and edit it manually, use the
-:binary:`~bin.mongodrdl` program to create a ``.drdl`` schema
-file and use the :option:`--schema <mongosqld --schema>` option
-when starting :binary:`~bin.mongosqld`. For more information on schema
-generation and data sampling, see :doc:`schema-configuration`.
+By default, :binary:`~bin.mongosqld` generates a default :doc:`data schema
+</schema-configuration>` and stores it in memory. To create a custom schema
```

Comment:
Better to use wording in the current doc to show the difference of approaches to avoid potential confusion:

> To create a custom schema instead and edit it manually,

Alternatively, if you prefer to create a custom schema file and edit it manually,